cmake/configure/configure_1_threads.cmake

```diff
@@ -138,6 +138,24 @@ MACRO(FEATURE_THREADS_FIND_EXTERNAL var)
   IF(TBB_FOUND)
     SET(${var} TRUE)
   ENDIF()
+
+  #
+  # TBB versions before 4.2 are missing some explicit calls to std::atomic::load
+  # in ternary expressions; these cause compilation errors in some compilers
+  # (such as GCC 8.1 and newer). To fix this we simply blacklist all older
+  # versions:
+  #
+  IF(TBB_VERSION_MAJOR LESS 4 OR
+      ((TBB_VERSION_MAJOR EQUAL 4) AND (TBB_VERSION_MINOR LESS 2)))
```

Comment:
There is `VERSION_LESS` that you could easily use here: https://cmake.org/cmake/help/v3.0/command/if.html

Reply:
Good catch. Fixed.